dev/DatePicker/DatePicker_themeresources.xaml

```diff
@@ -4,6 +4,7 @@
   xmlns:contract7Present="http://schemas.microsoft.com/winfx/2006/xaml/presentation?IsApiContractPresent(Windows.Foundation.UniversalApiContract,7)"
   xmlns:contract7NotPresent="http://schemas.microsoft.com/winfx/2006/xaml/presentation?IsApiContractNotPresent(Windows.Foundation.UniversalApiContract,7)"
   xmlns:contract6Present="http://schemas.microsoft.com/winfx/2006/xaml/presentation?IsApiContractPresent(Windows.Foundation.UniversalApiContract,6)"
+  xmlns:primitives="using:Microsoft.UI.Xaml.Controls.Primitives"
```

Comment:
If there isnt one already, can you add a page to the common styles page (or another page) and add a test to load that page and verify that the control loads and you able to invoke the flyout ?

Reply:
CommonTestStyles have a test for DatePicker and TimePicker and both of them click on their respective buttons to invoke the flyout